text/1016-deprecate-this-attrs-in-templates.md

```diff
@@ -0,0 +1,61 @@
+---
+stage: recommended
```

Comment:
Can't start in recommended?

Reply:
I copied this from the original RFC, apologies

Reply:
done